src/ISA/ISA/ArcTypes/CompositeHeader.fs

```diff
@@ -159,6 +176,16 @@ type CompositeHeader =
         | FreeText s when s.ToLower() = "derived data file" -> true
         | _ -> false   
 
+    /// <summary>
+    /// Is true if this Building Block type is a CvParamColumn.
+    ///
+    /// The name "CvParamColumn" refers to all columns with the syntax "Parameter/Factor/etc [TERM-NAME]".
```

Comment:
What about "Protocol Type" which is a short form for "Parameter [Protocol Type]"

Reply:
It also follows the maincolumn-tsr-tan schema

Reply:
Hmm, well it does return true for the `IsTermColumn`. I needed a functionality to specifically check for all the columns that have freely choosable categories.
![image](https://github.com/nfdi4plants/ISADotNet/assets/17880410/a62b46ed-0757-49f1-91b7-03ed4303d779)

Maybe I could add to the comment, that this does not return true for ProtocolType?